File: packages/functions/core/handler.py
import json
import requests

def main(event, context):
    """
    AWS Lambda handler function. SST's config will point to 'handler.main'.
    """
    response = requests.get('https://httpbin.org/get')
    
    return {
        "statusCode": 200,
        "body": json.dumps({
            "message": "Hello from Python Lambda!",
            "external_call_status": response.status_code
        })
    }

File: packages/functions/core/src/handler.ts
import { APIGatewayProxyEvent, APIGatewayProxyResult } from 'aws-lambda';

export const main = async (
  event: APIGatewayProxyEvent
): Promise<APIGatewayProxyResult> => {
  console.log('Hello from TypeScript Lambda!');

  return {
    statusCode: 200,
    headers: {
      'Content-Type': 'application/json',
      'Access-Control-Allow-Origin': '*',
      'Access-Control-Allow-Headers': 'Content-Type',
      'Access-Control-Allow-Methods': 'OPTIONS,POST,GET',
    },
    body: JSON.stringify({
      message: 'Hello from TypeScript Lambda!',
      event: event.httpMethod || 'Unknown method',
    }),
  };
};
